LICENSE

```diff
@@ -280,8 +280,6 @@ Apache Geode bundles the following files under the MIT license:
     Foundation and other contributors, http://jquery.org
   - jScrollPane (http://jscrollpane.kelvinluck.com/), Copyright (c) 2010
     Kelvin Luck
-  - matchMedia() polyfill (https://github.com/paulirish/matchMedia.js),
```

Comment:
Why is this removed?  I believe one of the js libraries that we include has bundled this in their source code.  See https://lists.apache.org/thread.html/ef3c70fcae89a05adbc46598439208a2e920d71b4f7062662a814fff%401460088338%40%3Cdev.geode.apache.org%3E.

Reply:
if it's in our source I should be able to find it in our source...I could not find any trace...

Reply:
The only reference to polyfill, which I tracked down from the email thread you shared, is in https://github.com/apache/geode-site/blob/master/website/content/js/respond.min.js

Since we don't distribute geode-site in source form as part of Geode releases, do we still need it in the Geode LICENSE?

Reply:
It looks like maybe geode-site used to be a module of geode back when polyfill was originally added to LICENSE.  Now geode-site is a separate repo and isn't shipped as part of Geode releases.  Coincidentally all the other javascript libraries other than polyfill are also used in geode-pulse, which we do ship.

Reply:
Nice work!  Thanks for filling in the gaps in my memory :-)